tasks/vhost.yml

```diff
@@ -3,6 +3,14 @@
 - name: ensure rabbitmq is runned
   service: name=rabbitmq-server state=started
 
+- name: delete rabbitmq vhost
+  rabbitmq_vhost: >
+    name="{{ item.name }}"
+    node="{{ item.node | default('rabbit') }}"
+    state=absent
+  with_items: "{{ rabbitmq_vhost_definitions }}"
+  when: "rabbitmq_force_vhost_recreate | default('no') | bool"
```

Comment:
Instead `rabbitmq_force_vhost_recreate` I would suggest `rabbitmq_vhost_force_recreate`, when variable will be used in alphabetical order it will be easier to link that variable to the vhost part.

What do you think?